templates/jlink/jlink-builder/jlink-builder-template.yaml

```diff
@@ -29,7 +29,15 @@ objects:
       dockerfile: |
         FROM -
         USER 0
-        RUN microdnf --setopt=install_weak_deps=0 --setopt=tsflags=nodocs install -y java-${JDK_VERSION}-openjdk-jmods
+        RUN mkdir -p /mnt/jrootfs
+        RUN microdnf install --installroot /mnt/jrootfs --releasever 9 --setopt install_weak_deps=0 --nodocs -y java-${JDK_VERSION}-openjdk-jmods \
```

Comment:
we don't need the jmods RPM content to be in the final image.

Reply:
ok then I retain the original line and add the remaining stuff necessary for /mnt/jrootfs

Reply:
Great -- that puts jmods back into the builder